_posts/2021-08-25-DWAngularMaterialDesignSolutions.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Utilising the Angular Material Library to Provide Bespoke Design Solutions"
+author: Daniel Warren
+categories:
+  - front-end-development
+  - ux-design
+tags:
+  - Angular
+  - UX
+  - GUI
+---
+
+
+The [Angular Material Library](https://material.angular.io/) is a collection of Material Design components which integrates with [Angular](https://angular.io/), a typescript based, open-source web development framework. The UI elements that the library provides are:
+
+- Versatile: the same elements can be used on a desktop or mobile browser.
+- Modifiable: elements can be adapted to suit unique design requirements.
+- Styleable: styles can be created and applied to all elements in a project, allowing for colour schemes to be lifted from a UX/UI Designer's prototype and applied during development.
+
+For these reasons Angular has become the foremost development framework within BGS Digital for application development, web development, and PWA (progressive web app) development.
+
+![Demo-Mobile](https://github.com/DanielLyleWilliamWarren/BlogDay_AngularMaterial/raw/main/assets/opacitysolution.gif)
+
+_Figure 1: Side navigation bar with bespoke solution to adjusting layer opacity in a mobile layout, the BGS colour scheme has been applied to all elements._
+
+The above graphic shows a design solution implemented to address the requirement for a 'Layer Options Panel'. This was achieved by using a collection of elements from the Material Library, some straight out of the box such as the side navigation panel which has been populated with BGS Explorer specific content, and the dropdown menus which host the 'Layer Options Panel'. Some material elements have been modified to fit the requirements of the design such as the 'Opacity Slider' which is a dropdown menu with certain functionalities hidden from the user. All elements however have the same 'BGS Explorer' specific styling applied.
```

Comment:
I think deleting 'and the dropdown menus which host the Layer Options Panel' will make it read better without losing anything.